File: check_users.go
package main

import (
	"fmt"
	"nofx/store"
)

func main() {
	st, err := store.New("data/data.db")
	if err != nil {
		fmt.Printf("Failed to connect to database: %v\n", err)
		return
	}
	defer st.Close()

	// Query all users
	var users []struct {
		ID          string `gorm:"column:id"`
		Email       string `gorm:"column:email"`
		OTPVerified bool   `gorm:"column:otp_verified"`
	}

	err = st.GormDB().Table("users").Find(&users).Error
	if err != nil {
		fmt.Printf("Failed to query users: %v\n", err)
		return
	}

	fmt.Println("现有用户:")
	fmt.Println("========")
	for _, user := range users {
		fmt.Printf("ID: %s\n", user.ID)
		fmt.Printf("Email: %s\n", user.Email)
		fmt.Printf("OTP Verified: %v\n", user.OTPVerified)
		fmt.Println("--------")
	}
}

File: check_users.py
import sqlite3

def check_users():
    """检查用户信息"""
    try:
        conn = sqlite3.connect('data/data.db')
        cursor = conn.cursor()
        
        print("=== 用户信息检查 ===")
        cursor.execute("SELECT email, created_at FROM users LIMIT 10")
        users = cursor.fetchall()
        
        if users:
            print("现有用户:")
            for user in users:
                print(f"  邮箱: {user[0]}, 创建时间: {user[1]}")
        else:
            print("无用户数据")
            
        conn.close()
    except Exception as e:
        print(f"查询用户信息出错: {e}")

if __name__ == "__main__":
    check_users()